app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show, :friends]
+  before_action :set_current_user, only: [:me, :edit]
+
+
+  def index
```

Comment:
Our `index` action seems to have a lot of "action". Please run me down on what exactly do we want our `index` action to do.